ra/ra.go

```diff
@@ -1152,12 +1160,32 @@ func (ra *RegistrationAuthorityImpl) validateFinalizeRequest(
 			ID:            authz.ID,
 			ChallengeType: solvedByChallengeType,
 		}
+
+		// Count the number of authorizations where the original CAA check was
+		// reused and the number where it was rechecked.
+		staleCAA, _ := validatedBefore(authz, ra.clk.Now().Add(caaRecheckDuration))
```

Comment:
The error is being thrown away, but on the next line you're checking for `err` which I don't think will be the err that's expected.

Reply:
Oh whoops, yeah good catch.